CHANGES.md

```diff
@@ -136,6 +134,19 @@ and the specification might change in the near future.
 [@planetarium/account]: https://www.npmjs.com/package/@planetarium/account
 
 
+Version 0.50.1
+--------------
+
+Released on March 6, 2023.
+
+ -  Ported changes from [Libplanet 0.49.1] and [Libplanet 0.49.2] release.
+    [#2884]
```

Comment:
```suggestion
    [[#2884]]
```
:eyes:

Reply:
Welp, guess this also needs to be fixed in 0.50-maintenance 🙃